convert2rhel/systeminfo.py

```diff
@@ -149,10 +151,30 @@ def _generate_rpm_va(self):
                          " minutes. It can be disabled by using the"
                          " --no-rpm-va option.")
         rpm_va, _ = utils.run_subprocess("rpm -Va", print_output=False)
-        output_file = os.path.join(logger.LOG_DIR, "rpm_va.log")
+        output_file = os.path.join(logger.LOG_DIR, log_filename)
```

Comment:
Test to check that it defaults to the global variable

Reply:
I believe there's no need to test the `log_filename`, because we accept any value, any value you put that will be used, even blank. If you put blank value, function `store_content_to_file` will be raise a exception and will close the file cursor as expected.

Reply:
As I mentioned in the other thread, seems like there are tests and I missed it.